src/System.Private.CoreLib/shared/System/Text/StringBuilder.cs

```diff
@@ -1663,65 +1661,36 @@ internal StringBuilder AppendFormatHelper(IFormatProvider provider, string forma
                         ch = format[pos];
                         pos++;
```

Comment:
Would it make the rest of the code a little simpler if this `pos++` moved past the `ch == '}'` check? It might make the rest of the diff align a little better? (i.e. no  `-1`, no new `else if`)

Reply:
Changed accordingly. Still changed the if to an else if though, no need to double check the curly. Eliminated the else and put the pos++ increment where it was before.